test/e2e/prune_test.go

```diff
@@ -101,4 +101,27 @@ var _ = Describe("Podman rm", func() {
 		Expect(len(images.OutputToStringArray())).To(Equal(0))
 	})
 
+	It("podman system prune pods", func() {
+		SkipIfRemote()
+
+		session := podmanTest.Podman([]string{"pod", "create"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+
+		session = podmanTest.Podman([]string{"pod", "start", "-l"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+
+		session = podmanTest.Podman([]string{"pod", "stop", "-l"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+
+		prune := podmanTest.Podman([]string{"system", "prune", "-f"})
+		prune.WaitWithDefaultTimeout()
+		Expect(prune.ExitCode()).To(Equal(0))
+
+		images := podmanTest.Podman([]string{"pod", "ps"})
+		images.WaitWithDefaultTimeout()
+		Expect(len(images.OutputToStringArray())).To(Equal(0))
+	})
```

Comment:
It would probably be good to have at least one running pod in the mix to make sure it survives.  Also in most of the checks you verifying that the status is good, but don't check to make sure the right number of pods remain.  It would be good to add that too, even if it's checking for zero pods.

Reply:
fixed!